.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: java
+jdk:
+  - oraclejdk8
+  - oraclejdk7
+branches:
+  only:
```

Comment:
It does test PRs - see details of this merge. This is to restrict random branch push testing.